packages/victory-canvas/src/curve.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { useCanvasContext } from "./hooks/use-canvas-context";
+import { LineHelpers } from "victory-core";
+
+const Curve = (props) => {
+  const { canvasRef, clear } = useCanvasContext();
+  const { style, data } = props;
+  const { stroke, strokeWidth } = style;
+
+  const draw = React.useCallback(
+    (ctx) => {
+      const line = LineHelpers.getLineFunction(props);
+      ctx.strokeStyle = stroke;
+      ctx.lineWidth = strokeWidth;
+      ctx.beginPath();
+      line.context(ctx)(data);
```

Comment:
Really wonderful that the d3 shape functions just work

Reply:
Yes!